api/swagger-spec/v1.json

```diff
@@ -13193,7 +13193,7 @@
       "type": "versioned.Event",
       "method": "GET",
       "summary": "watch individual changes to a list of Secret",
-      "nickname": "watchNamespacedSecretList",
+      "nickname": "watchNamespacedSecretListForAllNamespaces",
```

Comment:
I'm not sure how critical the first "Namespaced" is. E.g., maybe (watchNamespacedSecretList, watchNamespacedSecretListForAllNamespaces) could instead be (watchSecretList, watchSecretListForAllNamespaces)?

(It's in there to distinguish between cluster and namespace scoped resources. I'm not sure if we have any with colliding names any more. @deads2k?)